packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/optimize-ciemss-operation.ts

```diff
@@ -7,9 +7,9 @@ import { createInterventionPolicy } from '@/services/intervention-policy';
 const DOCUMENTATION_URL =
 	'https://github.com/ciemss/pyciemss/blob/main/pyciemss/interfaces.py#L747';
 
-export enum InterventionTypes {
-	paramValue = 'param_value', // provide a parameter value to get a better start time.
-	startTime = 'start_time' // provide a statr time to get a better parameter value.
+export enum OptimizationTypes {
+	getStartTime = 'start_time', // provide a parameter value to get a better start time.
+	getParamValue = 'param_value' // provide a statr time to get a better parameter value.
```

Comment:
We do not normally use verbs as enumeration names, is there a problem if we do "StartTime", "ParamValue"?

Reply:
I wasnt aware of that norm
I can update to the following if it is easier to read?
```
OptimizationInterventionObjective{ 
   startTime ...
   paramValue ...
}
```